app/models/job_invocation.rb

```diff
@@ -48,7 +48,7 @@ class JobInvocation < ActiveRecord::Base
 
   scope :with_task, -> { references(:task) }
 
-  scoped_search :in => :recurring_logic, :on => 'id', :rename => 'recurring_logic.id', :auto_complete => true
+  scoped_search :in => :recurring_logic, :on => 'id', :rename => 'recurring_logic.id', :complete_value => true
```

Comment:
I don't like it either but I think it's required because of this https://github.com/theforeman/foreman_remote_execution/blob/master/app/models/job_invocation_task_group.rb#L14